infrastructure/terraform/web-ui.tf

```diff
@@ -0,0 +1,26 @@
+module "web_ui_secrets" {
+  source = "modules/service-secrets"
+  project_name = "taskcluster-ui"
```

Comment:
isn't it called taskcluster-web?

Reply:
I don't want to call it taskcluster-web in Kubernetes, because it leads to names like "taskcluster-web-web" in there, so I decided to go with the dir name

Reply:
We already have a secrets_secrets in there somewhere iirc. May as well continue the trend. It's just nice to map it to the name of the package if we can.

Reply:
The package in the monorepo resides in the `ui` dir? I was hoping to have more distinguishable names than `web` and `web-server`

Reply:
I think a thorough renaming to `taskcluster-ui` could make sense, if @helfi92 is ok with it.

Reply:
I’m okay with both -web and -ui.